yt_dlp/YoutubeDL.py

```diff
@@ -3137,7 +3137,10 @@ def process_subtitles(self, video_id, normal_subtitles, automatic_captions):
             except re.error as e:
                 raise ValueError(f'Wrong regex for subtitlelangs: {e.pattern}')
         else:
+            # Prioritize original language subtitles (e.g., "it-orig") over translated ones
+            orig_lang = next((lang for lang in all_sub_langs if lang.endswith('-orig')), None)
             requested_langs = LazyList(itertools.chain(
+                [orig_lang] if orig_lang else [],
```

Comment:
this doesn't belong in core

Reply:
you mean the commit name or the location of the code itself?

Reply:
This change should not be made in core at all

Reply:
let me see if I can get around this

Reply:
simplified the fix and moved in `youtube` extractor. Auto-translations did returned `429` so I'm not sure how to get these.